pandas/plotting/_core.py

```diff
@@ -1603,7 +1603,8 @@ def scatter(self, x, y, s=None, c=None, **kwargs):
             The column name or column position to be used as vertical
             coordinates for each point.
         s : str, scalar or array-like, optional
-            The size of each point. Possible values are:
+            The size of each point. ``size`` is also accepted as an alternate keyword
```

Comment:
i think we can list `s, size : str....` here (i don't know if we do this for aliases elsewhere but should follow the same pattern), e.g. see read_csv where we have some aliases

Reply:
If I follow the syntax of `read_csv` I have to write

```
size : str, scalar or array-like, optional
    Alias for s
```

 I did not find a notation like `s, size: str`.